target/linux/ramips/image/mt7620.mk

```diff
@@ -331,6 +331,19 @@ define Device/edimax_br-6478ac-v2
 endef
 TARGET_DEVICES += edimax_br-6478ac-v2
 
+define Device/edimax_ew-7476hpc-16m
+  SOC := mt7620a
+  DEVICE_VENDOR := Edimax
+  DEVICE_MODEL := EW-7476HPC (CAP1200 16M)
```

Comment:
see above.

Reply:
This device is a 16M variant, others exist with 8M.